style/layer/poi.js

```diff
@@ -0,0 +1,82 @@
+import * as label from "../constants/label.js";
+
+const poiIcon = [
+  "match",
+  ["get", "class"],
+  "bar",
+  "martini-glass",
+  "beer",
+  "martini-glass",
+  "hospital",
+  "hospital-cross",
+  "dot_city",
+];
+
+const poiFilter = [
+  "any",
+  ["==", ["get", "class"], "bar"],
+  ["==", ["get", "class"], "beer"],
+  ["==", ["get", "class"], "hospital"],
+];
+
+const iconColor = [
+  "match",
+  ["get", "class"],
+  "bar",
+  "#0000dd",
+  "beer",
+  "#0000dd",
+  "hospital",
+  "#ff0000",
```

Comment:
The specific color values here could use some refinement to keep the icons from looking gaudy. As a starting point, the red for freeways should be _the_ red for anything other than a fill in this style:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/823df2da905d2fbe26b39273d814277f1043d1cc/style/layer/road.js#L235-L247